### DISCUSSION TITLE
Some questions and suggestions

### DISCUSSION THREAD

**Discussion by gyt95** [community/community#39291] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Questions:
1.How to collect a discussion if it is helpful for me?
2.If I remove all the recommendations like "Start writing code" in my dashboard. How can I make them appear again?(I really like these recommendations, but sometimes it prevents me from watching followers' dynamics)

Suggestions:
1.Users can see the specific like time of each repo they like.
2.Please add a page notification push. Sometimes I watch some important projects and submit a PR or issue, but can't receive other people's messages immediately, so I can only enter the message center to check, it's not convenient.
3.The official can have more animation-style teaching, which is friendly to newcomers.
4.Hope to provide a list or mark, you can see common followers with other users. This might not be necessary, but might be interesting.

**Comment by gyt95** [comment]
Maybe I should post in the right section?

**Comment by jfullstackdev** [comment]
> 1.How to collect a discussion if it is helpful for me?

simply bookmark it or watch it by subscribing

![image](https://user-images.githubusercontent.com/47092464/202845962-78a0bca5-ba83-49e0-8860-15b6a49d2649.png)

> 2.If I remove all the recommendations like "Start writing code" in my dashboard. How can I make them appear again?(I really like these recommendations, but sometimes it prevents me from watching followers' dynamics)

that's a starter info, so when you click it, it will not appear again, as far as I know, if you delete all the history of your browser, it will appear again

> 1.Users can see the specific like time of each repo they like.

not very sure what you mean by this 

> 2.Please add a page notification push. Sometimes I watch some important projects and submit a PR or issue, but can't receive other people's messages immediately, so I can only enter the message center to check, it's not convenient.

there is a notification for a repo, even a custom one, and as far as I know, it's also available at the mobile version

![image](https://user-images.githubusercontent.com/47092464/202846122-0baef95e-56c7-43c1-ab0b-2608c74ebd1b.png)

> 3.The official can have more animation-style teaching, which is friendly to newcomers

what official is that ? 

> 4.Hope to provide a list or mark, you can see common followers with other users. This might not be necessary, but might be interesting.

like, if you visit a certain user, there will be an info like common friends or followers ?

**Reply by gyt95** [reply]
I'm glad you were so patient in answering my questions and responding to my suggestions!
Maybe I need to explain a few points, I admit that sometimes my presentation is a bit problematic.

Suggestions
> 1. Users can see the specific like time of each repo they like.
> not very sure what you mean by this

This is actually a less necessary but somewhat interesting detail. For example, if I starred repo A today, I should be able to see the specific time of star this project next to repo A in my stars list. Because sometimes I review my stars list, and if I can see the specific time, I can know what I am focusing on in each period.

> 2.Please add a page notification push. Sometimes I watch some important projects and submit a PR or issue, but can't receive other people's messages Sometimes I watch some important projects and submit a PR or issue, but can't receive other people's messages immediately, so I can only enter the message center to check, it's not convenient.
> there is a notification for a repo, even a custom one, and as far as I know, it's also available at the mobile version

It's true that every repo has a notification. But it seems that I can only find out if there are new messages in the dashboard? If I'm looking at an item and I close the page after I'm done, I won't be able to know if there's a message in real time. (Mobile seems to be doing a better job at this point?)

> 3.The official can have more animation-style teaching, which is friendly to newcomers
> what official is that ?

The official refers to the Github team. In fact, in recent years Github has introduced features such as discussion to make the whole community richer. If the Github team could launch more related tutorials, more people could be involved in the community.

> 4.Hope to provide a list or mark, you can see common followers with other users. this might not be necessary, but might be interesting.
> like, if you visit a certain user, there will be an info like common friends or followers ?

Yes, yes! as you said. For example, a message on the user's home page somewhere showing who they are following in common.

Thanks again for your reply

**Reply by airtower-luna** [reply]
> If I'm looking at an item and I close the page after I'm done, I won't be able to know if there's a message in real time.

You can enable email notifications (see [Configuring notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications)), you'll receive those no matter if you have the website open. :slightly_smiling_face:

**Reply by gyt95** [reply]
> > If I'm looking at an item and I close the page after I'm done, I won't be able to know if there's a message in real time.
> 
> You can enable email notifications (see [Configuring notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications)), you'll receive those no matter if you have the website open. 🙂

Thank you very much!! although I'm not really used to checking emails.
